### DISCUSSION TITLE
install had a problem...

### DISCUSSION THREAD

**Discussion by Kooshin00** [Security-Onion-Solutions/securityonion#13387] [2.4] [answered]
### Version

2.4.80

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Eval

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

6

### RAM

16GB

### Storage for /

72GB

### Storage for /nsm

138GB

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

hello , i just installed the security onion inside vmware and followed the youtube installation guideline and didnt arrive at the web interface - install had a problem.
![so-eval-2024-07-24-16-04-40](https://github.com/user-attachments/assets/0aa20dbc-147b-4c1c-8c6e-b3c5c1c1baf2)

for /root/sosetup.log
![rootsosetup log](https://github.com/user-attachments/assets/a124a1fd-ca5c-4f0f-a114-6b3bd395325f)

for /root/errors.log
![Screenshot 2024-07-24 164501](https://github.com/user-attachments/assets/4c7809e1-4725-4f7b-bd2e-09f6f3fa6592)

am new to security onion and currently learning, any assistance would be greatly appreciated. thanks

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
What happens if you run `sudo so-checkin`? Does it complete successfully?

**Reply by Kooshin00** [reply]
yes all successful  no failure 
![so-checkin](https://github.com/user-attachments/assets/d54bee74-be06-4924-9696-e1c2114abd3e)

**Reply by cm-ops** [reply, marked as answer]
If the highstate completed without error you should be good.

**Reply by Kooshin00** [reply]
am not able to access on the web when i enter https://so-eval or the ip of the security onion , and am accessing from one of the ip range i allowed

**Reply by cm-ops** [reply]
Check these two commands: `sudo salt-call pillar.get global:url_base` and `sudo salt-call pillar.get firewall:hostgroups:analyst` You will see what your SOC address is and what IPs are allowed to connect via the host firewall. Check that IP/range against what your machine's IP is to see if it is allowed.
